### ISSUE TITLE
Value of tag containing properties is not rendered correctly

### ISSUE THREAD

**Issue by tbouchnafa** [michaelkourlas/node-js2xmlparser#33]
I have the following use case and want to have the value of an element as a string directly within the element: 
```js 
var x = require("js2xmlparser");
var options = {
        useCDATA: true
};
var result = {heading: {property: {"@": {value:"hi"}}}, "#": "test"};
var xml = x("tag", result, options);
console.log(xml); 
```

expected xml: 
```xml
<tag>
	<heading>
		<property value="hi"/>
                <?[CDATA[test]]>
	</heading>
</tag>
```
Running the code, I get: 
```xml
<tag>
	<heading>
		<property value="hi"/>
	</heading>
</tag>
```
Is this a wanted behavior, or is it possible to have the expected result?

**Comment by michaelkourlas** [comment]
This looks like a bug. I'll fix it shortly.

**Comment by michaelkourlas** [comment]
This is now fixed in 2.0.0:

```javascript
var options = {
    cdataKeys: ["*"]
};
var result = {
    heading: {property: {"__attr": {value: "hi"}}},
    "__val": "test"
};
var xml = js2xmlparser.parse("tag", result, options);
console.log(xml);
```
```xml
<?xml version='1.0'?>
<tag>
    <heading>
        <property value='hi'/>
    </heading>
    <![CDATA[test]]>
</tag>
```
